bonfire/Bonfire.php

```diff
@@ -84,6 +86,23 @@ private function setupMenus()
         $menus->createMenu('iconbar');
     }
 
+	/**
+	 * Creates any admin-required widgets so they're
+	 * available to use by any modules.
+	 */
+	private function setupWidgets()
+	{
+		$widgets =  service('widgets');
+
+		$widgets->createWidget(Stats::class, "stats");
+		$widgets->widget("stats")
+			->createCollection("users");
```

Comment:
Creating groups for users, etc in the main Bonfire file doesn't seem appropriate to me. Since those are specific to a module I think just having a `stats` collection is fine here. That leaves the door open for other widget types that would have their own collections. Then user users and groups modules would just need to add their widgets to the stats collection.

Reply:
You're absolutely right: I forgot to specify that I've added a bunch of useless collections just to show that it works similar to your menu management